deploy/prometheus/100-node-unschedulable.PrometheusRule.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-node-unschedulable
+    role: alert-rules
+  name: sre-node-unschedulable
```

Comment:
should we explicitly lock the namespace?

Reply:
@rogbas I didn't add it because none of the other rules in this repository specify. I think that's a defect though and that we ought to place these in `openshift-monitoring`. I'll add it for this rule and then separately add for the rest.